lms/djangoapps/bulk_email/tasks.py

```diff
@@ -100,11 +102,19 @@ def _get_recipient_queryset(user_id, to_option, course_id, course_location):
     Recipients who are in more than one category (e.g. enrolled in the course and are staff or self)
     will be properly deduped.
     """
-    if to_option not in TO_OPTIONS:
+    if to_option.isdigit():
+        if not GroupedQuery.objects.filter(id=int(to_option)).exists():
+            message = "Bulk email TO_OPTION query id {0} does not exist".format(to_option)
```

Comment:
Is there a reason you chose to use a positional replacement here (`{0}`) instead of the typical kwarg replacement (`query_id=to_option`)?